### ISSUE TITLE
Beta 1 - The ordinal 341 could not be located in the DLL [...] dlls\Qt5Network.dll

### ISSUE THREAD

**Issue by EyeDeck** [TanninOne/modorganizer#679]
Windows 10 64-bit, and a fresh install of Beta 1 into the default directory yields the following on (attempted) launch:
![untitled](https://cloud.githubusercontent.com/assets/2722970/15125819/d3719fb2-15fc-11e6-87d0-a3983dd7e6a4.png)
Qt5Network.dll does exist in that directory (MD5 242E2D18B5AE1AE4D79FBA81F5A30757).

**Comment by Toriko69** [comment]
Same here

**Comment by Dakraid** [comment]
Looks like it's an issue with the supplied Qt5Network.dll not with MO itself. You can take QT5.5's Qt5Network.dll and it works fine.

**Comment by EyeDeck** [comment]
I tried replacing the .dlls with the ones in Qt5.5.7z supplied with Beta 1 and got the same error, which is unsurprising, given that it's the same file. I also tried the Qt5Network.dll from Alpha 4 (MD5 5DDE8E313536E43A32179D1382313974) and get a different error instead:
![untitled2](https://cloud.githubusercontent.com/assets/2722970/15128739/94b676d0-160b-11e6-9843-274ff302c0e5.png)

**Comment by TanninOne** [comment]
Please try redownloading & reinstalling MO. I've just updated the installer but not 100% sure the problem is what I think it is. I assume the Qt5Network.dll is incompatible with an openssl you have installed. Because the Qt5Network with the checksum you posted definitively contains the  ordinal 341

**Comment by rdunlap** [comment]
That seems to have fixed it. I got the same error and re-downloaded it and now all is good.

Robert

**Comment by EyeDeck** [comment]
Redownloaded and reinstalled from newest release and was still getting the error, but the program would launch. For good measure I updated the only thing that I knew of that was both running and using OpenSSL (Apache), then rebooted, and now the program starts without any errors.

Thanks.

**Comment by GamerAmelli** [comment]
I also have this issue.  Only just downloaded and installed and this error popped up.  
File reported missing is in fact there in the folder.
I've installed outside of C drive onto another hard drive.

**Comment by darkanvas** [comment]
Same issue as GamerAmelli describes it. I tried sourcing the DLL file elsewhere as suggested and now I get a new error:

"The application was unable to start correctly (0xc000007b). Click OK to close the application."

**Comment by GamerAmelli** [comment]
Forgot to say I also tried to do as darkanvas did in sourcing another dll file but receive the same message as darkanvas.

I'm also on Win10 64bit.

**Comment by darkanvas** [comment]
Woke up this morning and decided to search for the 'official' release of QT 5.5 rather than trying to just grab the appropriate DLL from an external (but unknown / unofficial) source.

Downloaded the official QT open source installer from here:
http://www.qt.io/download-open-source/

Installed the recommended download: "Recommended download: Qt Online Installer for Windows"

During installation I only ticked QT 5.6 and QT 5.5 for installation, as well as leaving a black 'square' in the Tools checkbox (default QT tools installation).

Once installation was complete, I installed the MO2 Beta over the top of the MO2 Alpha without removing the alpha files first.

I tried to launch MO2 ... complete success. Now loads fine.

Hoping this will help others with the same issue.

**Comment by victorcain** [comment]
just tried what darkanvas said and it doesn't work for me.

**Comment by victorcain** [comment]
+TanninOne : Thanks! installing another OpenSSL package helped

**Comment by GamerAmelli** [comment]
+darkanvas : I would install Qt as per your suggestion, but 5.5 is showing as 15gb and the 5.6 as 19gb to install, so I cancelled the installer. Seems like a bit of a massive file size to install for all its components just to fix MO2.

**Comment by darkanvas** [comment]
That's certainly true. Additionally there's another problem with the method, which is that the installer required you to store temp files on your C:\ during installation. So not only do you need 15+ GB spare, it needs to be on your C:\ (at least temporarily).

I just about managed to scrape that together, luckily that space on the C:\ is not taken up post-install if you install to an external drive. I only tried this as I have colossal amounts of external storage, to which 15GB is nothing. If I had to keep the files installed on my C:\ permanently however, I might not have bothered

**Comment by Asshat** [comment]
Got the same issue, I re-installed MO2 & QT from official source. Nothing worked.

**Comment by TanninOne** [comment]
Is this still an issue in beta2?

**Comment by trentswd** [comment]
Just installed beta2 with the installer and got the same problem.
Trying to download qt5.5.7z now

**Comment by TanninOne** [comment]
What do you mean "override qt5.5.7z didn't work"? Did you get an error message extracting the file?

Could you please extract the attached file to your MO2 directory and run the .bat once, then send me the generated hashes.txt file?
[hashes.zip](https://github.com/TanninOne/modorganizer/files/299305/hashes.zip)

**Comment by trentswd** [comment]
@TanninOne 
Thanks for replying.
I meant extracting the archive to overwrite the files in dlls folder. No, the archive is fine, no error message while extracting.
[hashes.txt](https://github.com/TanninOne/modorganizer/files/299311/hashes.txt)
File is attached. It is not english though.

**Comment by TanninOne** [comment]
Odd. So everything seems to be installed correctly.
Please download and unzip this tool: https://technet.microsoft.com/en-us/sysinternals/bb896656.aspx
Then start MO, leave the error message open, then open a command line and run "c:\path\to\where\you\unzipped\listdll.exe modorganizer.exe > dlls.txt", then send me that dlls.txt

**Comment by trentswd** [comment]
thx very much. I know why the tool
I have some other programs with openssl and they write their path to PATH. So the qt network lib find the wrong openssl, instead of the ones in dlls folder.
Copy the openssl dlls outside with the mo exe, and everything is ok now.
[dlls.txt](https://github.com/TanninOne/modorganizer/files/299331/dlls.txt)
file is attached. Thank you very much for your patience.

**Comment by TanninOne** [comment]
Yeah, iCLS Client seems to be notorious for putting an ancient openssl into the path, but MO is set up so it should use the openssl-dlls included with it.

**Comment by GrantSP** [comment]
@TanninOne Do you want to have a FAQ added to the guide pointing to an OpenSSL installation? If so any specific version that you require?
